CONTRIBUTING.md

```diff
@@ -1,45 +1,57 @@
 # Contributing to Milvus Documentation
 
-Contributions to the technical documentation of Milvus are welcome from everyone. We strive to make the contribution process simple and straightforward. Up-to-date technical documentation can be found at [milvus.io](https://www.milvus.io/).
+Contributions to the Milvus documentation are welcome from everyone. The Milvus community strives to make the contribution process simple and straightforward. To do that, we create this page to walk you through the whole process step by step.
 
-### Workflow 
 
-Generally, we follow the "fork-and-pull" Git workflow.
 
-1. [**Fork**](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo) the repo on Github
-2. [**Commit**](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork) changes to your own branch
-3. Submit a [**pull request**](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-requests) so that we can review your changes
+> Up-to-date technical documentation can be found at [milvus.io](https://www.milvus.io/).
 
-Remember to [sync your forked repository](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo#keep-your-fork-synced) *before* submitting proposed changes upstream. Quickly fork repositories and start making contributions directly from [milvus.io](https://www.milvus.io/docs/install_milvus.md) using the *Edit* button located in the top right corner of most pages.
 
-### Issues
 
-Flag issues and request improvements using the links below:
+[TOC]
```

Comment:
TOC automation is only implemented on Milvus.io. Besides, I don't think we need TOC on this page.